docs/volunteering/all-committees-meeting.md

```diff
@@ -0,0 +1,45 @@
+# All Committees Meeting
+
+Once a month all of the committees have a meeting.
+This is normally held on the second Tuesday of the month.
+
+## Purpose
+
+This meeting is to help ensure smooth communications between the separate teams, and also the trustees.
+
+It is not intended that this meeting becomes the sole means of communication between teams, but rather that it helps bring to light stuff that may affect other teams and provide a forum to catch and resolve these.
+
+## Who can attend
+
+There should be at least one member of each teams' committee at each meeting.
+There will also be at least one trustee in attendance.
+
+Other volunteers are welcome at the invite of their committee, if the committee decide they have something to add.
+
+## Agenda
+
+The rough agenda for the all-committees meeting is as follows;
+* Run through updates from each committee, including successes and challenges.
+* Review tasks raised in previous meetings
+* Talking points (AOB)
+
+The minutes will be opened up for edit approximately 1 week prior to the meeting.
+The team updates and any talking points should be filled in by 24 hours prior to the meeting.
+Committees should also take this opportunity to review and update any tasks they have outstanding.
+
+24 hours prior to the meeting, the minutes will be changed to read-only [0].
+Committees are encouraged to read the committee updates from other teams during this time, as this will make the meeting go much more smoothly.
+If a team has a talking point to add after this cut off they should raise it to the trustees, who will add it if appropriate and there is time for discussion.
+
+The trustees will review the talking points for suitability during this 24 hours.
+A topic will only be removed if it is deemed to not be appropriate for discussion by all committees (this has yet to happen).
+
+During the meeting any questions around the updates and the talking points will be discussed, and any resolutions added to the minutes.
+
+After the meeting the minutes will be moved to the organisation folder.
```

Comment:
Can we link to this folder? Maybe also mention that it's readable by all volunteers?
```suggestion
After the meeting the minutes will be moved to the [organisation folder](https://drive.google.com/drive/folders/11Lsb-ooNoWHnQTQ__WoBg3guNOxBtwtF), which is accessible by all volunteers.
```